test/test_product_config.py

```diff
@@ -970,51 +1090,56 @@ def make_vis(
     )
 
 
-def make_vis_4ant() -> product_config.VisStream:
+def make_vis_4ant() -> VisStream:
```

Comment:
Is this a coincidental change, since `VisStream` is in the namespace anyway?

Reply:
Yes - happened to spot it and it bugged me.